puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/foreman-discovery-image.groovy

```diff
@@ -0,0 +1,49 @@
+pipeline {
+    agent { label 'admin && sshkey' }
+
+    options {
+        timestamps()
+        timeout(time: 3, unit: 'HOURS')
+        disableConcurrentBuilds()
+        ansiColor('xterm')
+    }
+
+    stages {
+        stage('Run FDI build') {
+            steps {
+                deleteDir()
+                script {
+                    job_parameters = [
+                        proxy_repository: env.proxy_repository,
+                        repoowner: env.repoowner,
+                        branch: env.branch,
+                    ]
+                    runCicoJob("foreman-discovery-image-build", job_parameters)
+                }
+            }
+        }
+
+        stage('Publish FDI build') {
+            steps {
+                script {
+                    base_dir = "/var/www/vhosts/downloads/htdocs/discovery"
+
+                    sshagent(['deploy-downloads']) {
+                        // delete old files in the target folder
+                        sh "ssh root@web02.theforeman.org 'mkdir -p ${base_dir}/${output_dir}/ ; rm -f ${base_dir}/${output_dir}/*' || true"
+
+                        // publish on web
+                        sh "scp fdi*tar fdi*iso root@web02.theforeman.org:${base_dir}/${output_dir}/"
+
+                        // create symlinks
+                        sh "ssh root@web02.theforeman.org 'pushd ${base_dir}/releases/ && rm -f latest; ln -snf \$(ls -t | head -n 1) latest; popd' || true"
```

Comment:
```suggestion
                        sh "ssh root@web02.theforeman.org 'ln -snf "$output_dir" ${base_dir}/releases/latest'"
```

Reply:
that's not the same ;)
`output_dir` can be `nightly`, which will result in no update of the latest *releases* change, or it can be `releases/<version>`, which will ;)